src/include/distributed/commands/utility_hook.h

```diff
@@ -63,7 +63,10 @@ typedef struct DDLJob
 } DDLJob;
 
 
-extern void multi_ProcessUtility(PlannedStmt *pstmt, const char *queryString,
+extern void multi_ProcessUtility(PlannedStmt * pstmt, const char *queryString,
```

Comment:
Guess this is 1 of the problems for the style checker

```suggestion
extern void multi_ProcessUtility(PlannedStmt *pstmt, const char *queryString,
```